pgjdbc/src/test/java/org/postgresql/test/jdbc2/AutoRollbackTestSuite.java

```diff
@@ -111,6 +116,7 @@ public String getSql(ReturnColumns cols) {
   }
 
   private final AutoSave autoSave;
+  private final CleanSavePoint cleanSavePoint;
```

Comment:
@davecramer , As far as I can tell, you create a field, but this field seems to be unused.